index.js

```diff
@@ -524,6 +524,18 @@ TChannelConnection.prototype.resetAll = function resetAll(err) {
     var self = this;
     if (self.closing) return;
     self.closing = true;
+
+    var inOpKeys = Object.keys(self.inOps);
+    var outOpKeys = Object.keys(self.outOps);
+
+    self.logger.info('resetting all connections', {
```

Comment:
consider warning if there is an err argument and assume that no err === clean shutdown and err === not clean shutdown.